examples/model_compress/pruning/config.yml

```diff
@@ -15,4 +15,4 @@ trialCommand: python3 basic_pruners_torch.py --nni
 trialConcurrency: 1
 trialGpuNumber: 0
 tuner:
-  name: grid
+  name: GridSearch
```

Comment:
Thank you~ By the way, you can also modify the English document `docs/en_US/Compression/AutoPruningUsingTuners.rst` line 67.

Reply:
ok